fixtures/webstudio-remix-netlify-edge-functions/.webstudio/data.json

```diff
@@ -192,7 +192,7 @@
   "page": {
     "id": "9di_L14CzctvSruIoKVvE",
     "name": "Home",
-    "title": "Home",
+    "title": "\"Home\"",
```

Comment:
is it really necessary to have quotes for string values when we know its a string? I had this question for settings bindings too

Reply:
at the time of writing? idk